<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotusRecvDataViewDialog</class>
 <widget class="QDialog" name="MotusRecvDataViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1317</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QwtPlot" name="qwtPlot1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot2">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>80</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot3">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>80</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QwtPlot" name="qwtPlot6">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>420</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="dataBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
